clientcompat/pycompat/clientcompat_pb2_twirp.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, server_address):
 
     def __make_request(self, body, full_method):
         req = Request(
-            url=self.__target + "/twirp" + full_method,
+            url=self.__target + full_method,
```

Comment:
Clients should be fine with this. The only thing required to keep backwards compatibility would be to change the base-URL (__target) to include the "/twirp" prefix 👍